handler/src/main/java/io/netty/handler/ssl/JdkSslServerContext.java

```diff
@@ -221,27 +223,28 @@ public JdkSslServerContext(File trustCertChainFile, TrustManagerFactory trustMan
         }
     }
 
-    JdkSslServerContext(X509Certificate[] trustCertChain, TrustManagerFactory trustManagerFactory,
+    JdkSslServerContext(X509Certificate[] trustCertCollection, TrustManagerFactory trustManagerFactory,
                         X509Certificate[] keyCertChain, PrivateKey key, String keyPassword,
                         KeyManagerFactory keyManagerFactory, Iterable<String> ciphers, CipherSuiteFilter cipherFilter,
                         ApplicationProtocolConfig apn, long sessionCacheSize, long sessionTimeout,
                         ClientAuth clientAuth) throws SSLException {
         super(ciphers, cipherFilter, toNegotiator(apn, true), clientAuth);
-        ctx = newSSLContext(trustCertChain, trustManagerFactory, keyCertChain, key,
+        ctx = newSSLContext(trustCertCollection, trustManagerFactory, keyCertChain, key,
                             keyPassword, keyManagerFactory, sessionCacheSize, sessionTimeout);
     }
 
-    private static SSLContext newSSLContext(X509Certificate[] trustCertChain, TrustManagerFactory trustManagerFactory,
-                                     X509Certificate[] keyCertChain, PrivateKey key, String keyPassword,
-                                     KeyManagerFactory keyManagerFactory, long sessionCacheSize, long sessionTimeout)
+    private static SSLContext newSSLContext(X509Certificate[] trustCertCollection,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method has 8 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS00107)